editor/js/Menubar.File.js

```diff
@@ -38,21 +38,67 @@ function MenubarFile( editor ) {
 	options.setClass( 'options' );
 	container.add( options );
 
-	// New
+	// New Project
 
 	var option = new UIRow();
-	option.setClass( 'option' );
-	option.setTextContent( strings.getKey( 'menubar/file/new' ) );
-	option.onClick( function () {
+	option.setClass("option");
```

Comment:
Please honor the code style of the project. E.g. `option.setClass("option");` should be:
```js
option.setClass( 'option' );
````
It's actually best if you configure your IDE/editor so it automatically detects the eslint settings ofthe repository.

Reply:
Sorry, Pettier plugin with bad configuration